### ISSUE TITLE
Configurable map height on issues list view's expanded map mode

### ISSUE THREAD

**Issue by sanak** [gtt-project/redmine_gtt#104]
**Problem**
Currently, map height is fixed, but sometimes larger map size is better.

**Improvement**
Make configurable map height with HTML text-expander right-bottom grip, or plugin settings.

**Comment by dkastl** [comment]
Good point, and I must admit I had no good idea yet how to do this the best way. There are also the limitations of Redmine how you could style a page.

Some things I thought of:
- Button to toggle to map in fullscreen or as modal window
- (Option to) move the map to the sidebar (would require the fullscreen or modal window mode)
- 2-column issue page, with map one side map and other side form fields (but this could very hard to achieve)

There is also a problem eventually when using a mobile browser with small screen.
